CHANGES.md

```diff
@@ -1,3 +1,61 @@
+### Unreleased
+
+#### Parser (2021-01-??)
+
+Replace Earley by Menhir, Pratter and Sedlex
+
+**Syntax modifications:**
+
+- The semi colon as termination token has been added. Hence, a symbol is
+  declared with symbol `foo: bar;`.
```

Comment:
Perhaps it would be better to say that, now, a semi-colon must terminate every command and every tactic as well.